Week3/homework/1-step3.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 function foo(func) {
-  // What to do here? 
+  func();
 }
 
```

Comment:
I would suggest returning the results of calling `func()`.
`foo` does not know anything about func, just that it needs to call it. 
So, it should be using (i.e. returning) the results of the function call.